lib/ui/src/components/sidebar/Heading.tsx

```diff
@@ -42,6 +42,7 @@ const SkipToCanvasLink = styled(Button)(({ theme }) => ({
   '@media (min-width: 600px)': {
     display: 'block',
     position: 'absolute',
+    top: 0,
```

Comment:
Can you please change this to `top: 20px` and `left: 20px`?

Reply:
Done.